tests/SpaceTime/GravitationalConstantTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using Bearded.Utilities.SpaceTime;
+using Bearded.Utilities.Tests.Helpers;
+using FluentAssertions;
+using FsCheck.Xunit;
+using OpenTK;
+using Xunit;
+
+namespace Bearded.Utilities.Tests.SpaceTime
+{
+    public class GravitationalConstantTests
+    {
+        public class TheNumericValueProperty
+        {
+            [Property]
+            public void ReturnsTheValuePassedToTheConstructor(float value)
+            {
+                var g = new GravitationalConstant(value);
+
+                g.NumericValue.Should().Be(value);
+            }
+        }
+
+        public class TheZeroProperty
+        {
+            [Fact]
+            public void ReturnsAZeroValue()
+            {
+                var zero = GravitationalConstant.Zero;
+
+                zero.NumericValue.Should().Be(0);
+            }
+        }
+
+        public class TheOneProperty
+        {
+            [Fact]
+            public void ReturnsAZeroValue()
+            {
+                var one = GravitationalConstant.One;
+
+                one.NumericValue.Should().Be(1);
+            }
+        }
+
+        public class TheAccelerationTowardsWithDifference2Method
+        {
+            [Property(Arbitrary = new []{typeof(LimitedRangeFloatGenerator)})]
+            public void CalculatesAccelerationCorrectly(
+                float gValue, float massValue, float differenceXValue, float differenceYValue)
+            {
+                var g = new GravitationalConstant(gValue);
+                var mass = new Mass(massValue);
+                var difference = new Difference2(differenceXValue, differenceYValue);
+                var expectedAcceleration = gValue * massValue
+                    * new Vector2(differenceXValue, differenceYValue).Normalized()
```

Comment:
nit: I would put the vector aspect of this calculation either at the start or the front to leave the numeric part of it intact. I think that would make it more recognisable as `Gm / r^2`.
Also consider extracting a `radiusSquared` variable

(Same below with the unit one)